packages/graphql/lib/schema-builder/storages/type-metadata.storage.ts

```diff
@@ -216,20 +216,30 @@ export class TypeMetadataStorageHost {
   }
 
   loadClassPluginMetadata(metadata: ClassMetadata[]) {
+    const seen = new Set<Function>();
```

Comment:
nit: not sure if `seen` is descriptive enough for the purpose it serves

Reply:
How about `loadedClasses`? Or maybe`handled Classes`?

Reply:
`loadedClasses` sounds good